Formula/etcher-cli.rb

```diff
@@ -0,0 +1,32 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher/archive/v1.4.4.tar.gz"
+  sha256 "02082bc1caac746e1cdcd95c2892c9b41ff8d45a672b52f8467548cad4850f5d"
+
+  depends_on "node@6" => :build # must be abi compatiable to the node version used with pkg
+  depends_on "python" => :build
+
+  def install
+    ENV["NODE_ENV"] = "production"
+    system "npm", "install", *Language::Node.local_npm_install_args
```

Comment:
I know I wrote this, but the above 2 lines could be unified as:
```
system "npm", "install", *Language::Node.local_npm_install_args, "--production"
```

Reply:
Nope, `--production` just tells `npm` to not install `devDependencies` if used without a package specified and does not have any effect if installing a specific package. (We use this to avoid installing all the electron tooling)

Reply:
... and of cause it didn't work because of their old `npm-shrinkwrap.json`file. We have to remove this file first.